src/crypto/crypto_keys.cc

```diff
@@ -1134,6 +1136,51 @@ void KeyObjectHandle::InitEDRaw(const FunctionCallbackInfo<Value>& args) {
   args.GetReturnValue().Set(true);
 }
 
+void KeyObjectHandle::Equals(const FunctionCallbackInfo<Value>& args) {
+  std::shared_ptr<KeyObjectData> key =
+    Unwrap<KeyObjectHandle>(args.Holder())->Data();
+  std::shared_ptr<KeyObjectData> key2 =
+    Unwrap<KeyObjectHandle>(args[0].As<Object>())->Data();
+
+  KeyType keyType = key->GetKeyType();
```

Comment:
snake_case for local variables.

```suggestion
  KeyType key_type = key->GetKeyType();
```